src/jit/valuenum.cpp

```diff
@@ -5858,8 +5916,9 @@ void Compiler::fgValueNumberTree(GenTreePtr tree, bool evalAsgLhsInd)
                                     // Now get rid of any remaining struct field dereferences. (if they exist)
                                     if (fldSeq->m_next)
                                     {
-                                        storeVal = vnStore->VNApplySelectorsAssign(VNK_Liberal, fldMapVN,
-                                                                                   fldSeq->m_next, storeVal, indType, compCurBB);
+                                        storeVal =
+                                            vnStore->VNApplySelectorsAssign(VNK_Liberal, fldMapVN, fldSeq->m_next,
+                                                                            storeVal, indType, compCurBB);
```

Comment:
There quite a few of these reformatting changes, that I would not have expected to see